resources/shellHelperFunctions.sh

```diff
@@ -257,7 +257,10 @@ set_npm_version () {
   # release of Keyman Developer -- these two versions should be in sync. Because this
   # is a large repo with multiple projects and build systems, it's better for us that
   # individual build systems don't take too much ownership of git tagging. :)
-  npm --no-git-tag-version --allow-same-version version "$version" || fail "Could not set package version to $version."
+  pushd . > /dev/null
```

Comment:
Should this fail if a version isn't passed in l.254?

`npm run version` results in

> lerna version --amend --yes --force-publish --no-git-tag-version

and waits at a prompt:
> ? Select a new version (currently 14.0.225) (Use arrow keys)
> Patch (14.0.226)
  Minor (14.1.0)
  Major (15.0.0)
  Prepatch (14.0.226-alpha.0)
  Preminor (14.1.0-alpha.0)
  Premajor (15.0.0-alpha.0)
  Custom Prerelease
  Custom Version